packages/frontend/src/App.vue

```diff
@@ -306,7 +265,7 @@ export default {
       this.rpc.invoke("executeCommand", [command, params]);
     },
     back() {
-      this.gotoStep(1); // go 1 step back
+      return this.gotoStep(1); // go 1 step back
```

Comment:
_.debounce ?

Reply:
no, back works different. The problem was with Next. I solved it